integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -40,7 +40,7 @@
 import tachyon.heartbeat.HeartbeatScheduler;
 import tachyon.job.CommandLineJob;
 import tachyon.job.JobConf;
-import tachyon.master.lineage.meta.LineageFileState;
+import tachyon.proto.JournalEntryProtos.LineageFileState;
```

Comment:
The enum is moving to protos because it needs to be serialized in the `LineageEntry` journal entry. We could potentially have two enums, one in protos and one in master.lineage.meta, but that's harder to maintain.